miqa/core/conversion/import_export_csvs.py

```diff
@@ -72,29 +85,38 @@ def validate_import_dict(import_dict, project: Optional[Project]):
 
 
 def import_dataframe_to_dict(df):
-    if list(df.columns) != IMPORT_CSV_COLUMNS:
+    if list(df.columns) != IMPORT_CSV_COLUMNS and list(df.columns) != IMPORT_CSV_COLUMNS[:6]:
```

Comment:
This should at least have a code comment explaining that there are two valid CSV formats.

Reply:
Added in [0ed474f](https://github.com/OpenImaging/miqa/pull/392/commits/0ed474f3cad06e516fd43dc99574db1cc8a55d7f)